lte/gateway/c/session_manager/SessionState.h

```diff
@@ -732,6 +734,7 @@ class SessionState {
  private:
   std::string imsi_;
   std::string session_id_;
+  int shard_id_;
```

Comment:
when is this initialized with the actual value of the shard (not 0)

Reply:
It's initialized when calling init session in local enforcer. I pass the shard id in through there.